wg-ai-gateway/charter.md

```diff
@@ -0,0 +1,94 @@
+# WG AI Gateway Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter
+README] and uses the Roles and Organization Management outlined in
+[wg-governance].
+
+[wg-governance]:https://github.com/kubernetes/community/blob/master/committee-steering/governance/wg-governance.md
+[Kubernetes Charter README]:https://github.com/kubernetes/community/blob/master/committee-steering/governance/README.md
+
+## Scope
+
+The AI Gateway Working Group focuses on the intersection of AI and
+networking, particularly in the context of extending load-balancer, gateway
+and proxy technologies to manage and route traffic for AI Inference.
```

Comment:
I’m wondering if this WG has some overlap with WG Serving. 

https://github.com/kubernetes/community/blob/bfee8f77de8c0e31c31cf7f90792148bd7be1f07/wg-serving/charter.md?plain=1#L37-L38 

Could you help clarify the scope differences?

Reply:
In this case it's squarely focused on the networking aspects of inference, not the compute or lifecycle management aspects as in the case of wg-serving. The key point is this one:

```
manage and route traffic for AI Inference.
```

Reply:
Right, but why the existing WG Serving isn't the right place to cover those topics? The biggest issue as I'm seeing is that you will affect WG Serving work, only if focusing on the networking side of it. So either explicitly listing out the reasons for that extraction or how are you going to collaborate with WG Serving in that scope will be necessary to ensure the work doesn't diverge.

Reply:
> So either explicitly listing out the reasons for that extraction or how are you going to collaborate with WG Serving in that scope will be necessary to ensure the work doesn't diverge.

I've added WG Serving as an explicit collaborator which we will need to keep looped in for review on proposals.

/cc @ArangoGutierrez @SergeyKanzhelev @terrytangyuan